storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.14.rst

```diff
@@ -0,0 +1,53 @@
+================================================================================
+|Percona XtraBackup| |release|
+================================================================================
+
+Percona is glad to announce the release of |Percona XtraBackup| |release| on
+|date|. Downloads are available from our `download site
+<http://www.percona.com/downloads/XtraBackup/Percona-XtraBackup-2.4.13/>`_ and
+from :ref:`apt <apt_repo>` and :ref:`yum <yum_repo>` repositories. This release
+introduces the suppport of Red Had Enterprise Linux 8.
+
+|Percona XtraBackup| |release| enables saving backups to an Amazon S3 storage
+when using |xbcloud|. The following example demonstrates how to use an Amaxon S3
+storage to make a full backup:
+
+.. code-block:: bash
+
+   $ xtrabackup --backup --stream=xbstream --extra-lsndir=/tmp --target-dir=/tmp | \
+   xbcloud put --storage=s3 \
+   --s3-endpoint='s3.amazonaws.com' \
+   --s3-access-key='YOUR-ACCESSKEYID' \
+   --s3-secret-key='YOUR-SECRETACCESSKEY' \
+   --s3-bucket='mysql_backups'
+   --parallel=10 \
+   ${date}-full_backup
+
+All Percona software is open-source and free.
+
+New Features
+================================================================================
+
+- Amazon S3 is now supported in |xbcloud|. More information in
+  :jirabug:`PXB-1813`.
+
+Bugs Fixed
+================================================================================
+
+- When the row format was changed, the server could crash during the incremental
```

Comment:
when row format was change *during the backup*, *xtrabackup* could crash during incremental prepare stage